app/configuration/Config.scala

```diff
@@ -103,7 +103,10 @@ object Config {
     MembershipRatePlanIds.fromConfig(ProductFamilyRatePlanIds.config(Some(config))(env, Membership))
 
   def demoPromo(env: String) = {
+    val jellyFishPromoCode = PromoCode("DGA85")
     val prpIds = digipackRatePlanIds(env)
+    val promoCodes = (Seq(jellyFishPromoCode) /: (88 to 94))((seq, i) =>  seq :+ PromoCode(s"DGA$i") :+ PromoCode(s"DGB$i"))
```

Comment:
So while I don't have the best record when it comes to avoiding obscure operators I found this quite tricky to read, it is a bit odd to fold a list and get a list back, you'd usually use some variant of map / flatMap for that.

I'd expect to see something more like this:
```scala
Seq(PromoCode("DGA85")) ++ (88 to 94).flatMap(n => Seq(PromoCode(s"DGA$n"), PromoCode(s"DGB$n")))
```

Though maybe that doesn't look much better after all